sdk/monitor/monitor-opentelemetry-exporter/test/internal/logUtils.spec.ts

```diff
@@ -98,6 +99,7 @@ describe("logUtils.ts", () => {
     attributes: {
       "some-attribute": "some attribute value",
       [ATTR_CLIENT_ADDRESS]: "127.0.0.1",
+      [experimentalOpenTelemetryValues.SYNTHETIC_TYPE]: "bot",
```

Comment:
nit: include a test that includes "non" supported value (unless we change to the "presence" as I mentioned) in which case we would want to test empty / whitespace strings as well.